server/routes/file.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const router = express.Router();
+const uploadImage = require("../utilities/uploadImage");
+
+//--File Endpoints (CRUD)--
+
+// update profile image
```

Comment:
Better documentation is prob needed for each endpoint we have. If u can add that example S3 request here with descriptions for params and desc. for what the endpoint does